### PR TITLE
[E084-MG < T0541-MG] Windows support

### PR DESCRIPTION
**Pull request by antaljanosbenjamin**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,3 +28,43 @@ jobs:
...
+          msystem: MINGW${{ matrix.arch.mingw }}
+          update: true
+          install: git mingw-w64-${{ matrix.arch.msys }}-toolchain mingw-w64-${{ matrix.arch.msys }}-cmake mingw-w64-${{ matrix.arch.msys }}-openssl
+      - name: Add mingw${{ matrix.arch.mingw }} to PATH
+        run: |
+          # First make sure python would resolve to the windows native python, not mingw one
+          echo "C:/msys64/mingw${{ matrix.arch.mingw }}/bin" | Out-File -Append -FilePath $env:GITHUB_PATH -Encoding utf8
+          echo "${{ env.pythonLocation }}" | Out-File -Append -FilePath $env:GITHUB_PATH -Encoding utf8
+      - name: Remove chocolatey gcc
+        run: choco uninstall mingw -y
```
Comment (antonio2368) [inline .github/workflows/ci.yml:63]: What's this? Can you add a comment with explanation?

File: setup.py [hunk 2]
```diff
@@ -154,7 +194,8 @@ def build_mgclient_for(self, extension: Extension):
             self.spawn([cmake_binary,
                         '--build', mgclient_build_path,
                         '--config', build_type,
-                        '--target', 'install'])
+                        '--target', 'install',
+                        '--verbose'])
```
Comment (antaljanosbenjamin) [inline setup.py:198]: The verbose output supposed to be temporary only, but the command always run in verbose mode ('self.verbose` is always not `None`), even when the verbosity is turned off with `--quiet`. Therefore I couldn't find a way in which the user can ask for verbose output, as a result the decision about use verbose output always or never. I choose to use it always, as the more is better in this case in my opinion.

**Comment by gitbuda** [comment]
> Yes, I wasn't totally sure about it, but couldn't think a better naming for that. Now I changed `has_more` to `has_more_out` and `my_has_more` to just `has_more`. If you have better idea, please let me know.

The current naming seems great!

**Review by gitbuda — APPROVED** [review]
Looks good :smile:

**Pull request merged**